kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -53,13 +54,56 @@ grCollectionsPanel.controller('GrNodeCtrl', ['collections', function(collections
 
 }]);
 
+grCollectionsPanel.controller('GrAddToCollectionCtrl',
+    ['mediaApi', '$scope', function(mediaApi, $scope) {
+
+    const ctrl = this;
+
+    ctrl.saved = false;
```

Comment:
But we should have some, I will work with the state a little more once I can figure out how. I might scrap the `ctrl` all together.